sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -557,3 +559,61 @@ def upsert_entity(  # pylint:disable=R1710
             return _trim_service_metadata(metadata)
         except HttpResponseError as error:
             _process_table_error(error)
+
+    def create_batch(
+        self,
+        **kwargs # type: Dict[str, Any]
+    ):
+        # type: (...) -> TableBatchOperations
+        """Create a Batching object from a Table Client
+
+        :return: Object containing requests and responses
+        :rtype: TableBatchOperations
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_batching.py
+                :start-after: [START batching]
+                :end-before: [END batching]
+                :language: python
+                :dedent: 8
+                :caption: Using batches to send multiple requests at once
+        return: Table batch operation for inserting new operations
+        rtype: ~azure.data.tables.TableBatchOperations
```

Comment:
These are duplicates of the :return: and :rtype: above.
Additionally, I don't think `TableBatchOperations` was exposed in the __init__ file, so this link wouldn't work.

Reply:
Same comment as the async one - these should be above the admonition with an additional ":" character.